basic-features.md

```diff
@@ -4,77 +4,125 @@ summary: Learn about the basic features of TiDB.
+| [Encryption at rest (TDE)](/encryption-at-rest.md)                                   | Y     | Y   | N   | N   | N   |
+| [Role-based authentication (RBAC)](/role-based-access-control.md)                    | Y     | Y   | Y   | Y   | N   |
+| [Certificate-based authentication](/certificate-authentication.md)                   | Y     | Y   | Y   | Y   | N   |
+| Support for MySQL 8.0 clients                                                        | Y     | Y   | N   | N   | N   |
+| [MySQL compatible `GRANT` system](/privilege-management.md)                          | Y     | Y   | Y   | Y   | Y   |
+| Auditing                                                                             | Y     | Y   | Y   | Y   | Y   |
+| **Data import and export**                                                           | **Development** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [Fast Importer (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md)         | Y     | Y   | Y   | Y   | Y   |
+| [mydumper logical dumper](/mydumper-overview.md)                                     | Deprecated | Deprecated | Y   | Y   | Y   |
+| [dumpling logical dumper](/dumpling-overview.md)                                     | Y     | Y   | N   | N   | N   |
+| [Transactional `LOAD DATA`](/sql-statements/sql-statement-load-data.md)              | Y     | N   | N   | N   | N   |
+| [Database migration toolkit (DM)](/migration-overview.md)                            | Y     | Y   | Y   | Y   | Y   |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                                  | Deprecated | Deprecated | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                                | Y     | Y   | N   | N   | N   |
+| **Management, observability and tools**                                              | **Development** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [TiDB Dashboard](/dashboard/dashboard-intro.md)                                      | Y     | Y   | N   | N   | N   |
+| [SQL diagnostics](/information-schema/information-schema-sql-diagnostics.md)         | Experimental | Experimental | N   | N   | N   |
+| [Information schema](/information-schema/information-schema.md)                      | Y     | Y   | Y   | Y   | Y   |
+| [Metrics schema](/metrics-schema.md)                                                 | Y     | Y   | N   | N   | N   |
+| [Statements summary tables](/statement-summary-tables.md)                            | Y     | Y   | N   | N   | N   |
+| [Slow query log](/identify-slow-queries.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| [TiUP deployment](/tiup/tiup-overview.md)                                            | Y     | Y   | N   | N   | N   |
+| [Ansible deployment](/online-deployment-using-ansible.md)                            | Deprecated | Deprecated | Y   | Y   | Y   |
```

Comment:
I don't think Ansible deployment is deprecated in v4.0 and the deployment version.  @zz-jason @kissmydb Please help confirm, thanks~

Reply:
Yeah, ansible deployment is deprecated since v4.0, we recommend users to use TiUP.

Reply:
@IANTHEREAL @lonng PTAL as well.

Reply:
v4.0 and the dev version still have the [Ansible deployment document](https://docs.pingcap.com/tidb/stable/online-deployment-using-ansible). The warning says: "Since v4.0, it is not recommended to deploy TiDB using TiDB Ansible." 

I guess Ansible is **NOT recommended** does not mean users cannot use it. If it is really deprecated, we need to remove all Ansible deployment-related content from v4.0 and the dev documentation.

Reply:
I think it is important to first deprecated, but keep the content. This tells users that they can keep using it for now, but should plan to switch before the next major version.

The note on the ansible page should use the consistent wording "deprecated" though, since not recommended is ambiguous. It could mean that it is "not recommended yet" (and in development) for example.

Reply:
As confirmed with @lucklove, `deprecated` here means that we no longer provide support for using Ansible to deploy TiDB 4.0. Although users can still use it forcibly, we are strongly against such usage. So `deprecate` is here appropriate.

Reply:
OK. I will resolve this conversation, since it is correct in this context.